### ISSUE TITLE
Dark mode as a free feature?

### ISSUE THREAD

**Issue by ezxpro** [standardnotes/app#1161]
I mean, having dark mode being a paid feature is a huge kick in the balls, don't you guys think?

I'm actually a paying notion user and I just began moving exploring standard notes so I move my sensitive data into it because of encryption.

The problem, I downloaded it 1 hour ago and it already hurts my eyes to look into this shiny white interface. I saw you have a config for dark mode under Appearance > Use system color scheme, but guess what?  Ha ha ha It's a 🎉🎉🎉 premium feature 🎉🎉🎉!
Who's the clown!?

Meeeee, who'll have to shell out $ 50 a month to test this without burning my retina

C'mon guys, this gotta be a joke right?
No one is offering dark mode as a premium feature anywhere, not that I know at least. Nowadays that's a basic feature of anything you'll keep staring at for a considerable period of time.

**Comment by Nicicalu** [comment]
You can buy a simple VPS Server and setup your own standardnotes server where you don't need a subscription to use any of these features.
You'd need a bit of linux, docker knowledge for this though...

**Comment by ezxpro** [comment]
Gosh, I uninstalled this already.

**Comment by Nicicalu** [comment]
> @Nicicalu Gosh, I uninstalled this already.

Yeah, well then why do you expect it to be free? They need to make money somehow...

**Comment by shompoe** [comment]
The mobile app has a built in dark theme for free users. And you can install a custom theme for free to use on any platform. Search for the awesomestandardnotes repository on GitHub for some community themes or make your own. Be aware though that not all the themes in that repository are being maintained.

**Comment by julianfairfax** [comment]
> The mobile app has a built in dark theme for free users. And you can install a custom theme for free to use on any platform. Search for the awesomestandardnotes repository on GitHub for some community themes or make your own. Be aware though that not all the themes in that repository are being maintained.

Are there any themes that actually are being maintained? It seems most of the entries in the awesome repo are all unmaintained.

**Comment by julianfairfax** [comment]
> > > The mobile app has a built in dark theme for free users. And you can install a custom theme for free to use on any platform. Search for the awesomestandardnotes repository on GitHub for some community themes or make your own. Be aware though that not all the themes in that repository are being maintained.
> > 
> > 
> > Are there any themes that actually are being maintained? It seems most of the entries in the awesome repo are all unmaintained.
> 
> I think for standard notes you either pay or you move on. Free version is basically a text editor with cloud synchronization. There are better alternatives out there if you dont want to buy standard notes.

I use https://github.com/arturolinares/sn-rme but I had to modify the package file to install it due to #1159 and it doesn't work in dark mode (this is not Standard Notes' fault though). Also of course long load times #1107 for both the app and the editor.

I tried self-hosting locally but that doesn't work either because of #1165. So it seems I'd have to pay for it if I wanted any maintained editors or themes. Alternatives aren't better either, I've tried them.

**Comment by shompoe** [comment]
I overhauled my theme recently but the white in the left hand pane which has come back again since I last did this still persists. This is because the current version of SN doesn't properly support custom themes. I confirmed this with support and they said it got hard coded as part of a fix for something. My theme is up to date with the latest specification and I am just waiting for it to start working as soon as they resolve that.

So even if they are maintained they won't work anyway.

--- 
Mark Desrousseaux



10 Aug 2022, 15:54 by ***@***.***:
